src/server/pipeline/match_data.py

```diff
@@ -59,47 +56,50 @@ def start(connection, added_or_updated_rows, manual_matches_df, job_id):
             log_db.log_exec_status(job_id, 'matching', 'executing', str({'at_row': row_num + 1, 'of_rows': len(rows)  }) )
 
         # Exact matches based on specified columns
-        row_matches = pdp_contacts[
-            (
-                (((pdp_contacts["first_name_normalized"] == row["first_name_normalized"]) &
-                (pdp_contacts["last_name_normalized"] == row["last_name_normalized"]))
-                |
-                ((pdp_contacts["first_name_normalized"] == row["last_name_normalized"]) &
-                (pdp_contacts["last_name_normalized"] == row["first_name_normalized"])))
-                &
-                ((pdp_contacts["email_normalized"] == row["email_normalized"]) | (pdp_contacts["mobile"] == row["mobile"]))
-            )
-        ]
+        query = text("select matching_id from pdp_contacts where archived_date is null and ( \
```

Comment:
If you triple-quote (""" """) the query strings you don't have to worry about the line breaks.

Reply:
good tip